### PR TITLE
setup E2E test in drone CI

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
This PR sets up E2E test in drone CI.

Part of: https://github.com/owncloud/web-app-dicom-viewer/issues/11

### CODE REVIEW

File: .drone.star [hunk 1]
```diff
@@ -1,9 +1,15 @@
 OC_CI_NODEJS = "owncloudci/nodejs:18"
 OC_CI_BUILDIFIER = "owncloudci/bazel-buildifier:latest"
 SONARSOURCE_SONAR_SCANNER_CLI = "sonarsource/sonar-scanner-cli:5.0"
+OC_CI_WAIT_FOR = "owncloudci/wait-for:latest"
+OCIS_IMAGE = "owncloud/ocis:latest"
```
Comment (saw-jan) [inline .drone.star:5]: we should pin ocis version
```suggestion
OCIS_IMAGE = "owncloud/ocis:5.0.0-rc.6" # or 5.0.0 when it gets released
```

File: .drone.star [hunk 2]
```diff
@@ -89,3 +95,112 @@ def installPnpm():
...
+                "pnpm build:w",
+            ],
+        },
+        {
+            "name": "wait-for-extension",
+            "image": OC_CI_WAIT_FOR,
+            "commands": [
+                "wait-for -it extension:9999 -t 300",
+            ],
+        },
```
Comment (saw-jan) [inline .drone.star:124]: wait can be removed
```suggestion
```

File: .drone.star [hunk 3]
```diff
@@ -89,3 +95,112 @@ def installPnpm():
...
+    ]
+
+def installPlaywright():
+    return [{
+        "name": "playwright-install",
+        "image": OC_CI_NODEJS,
+        "environment": {
```
Comment (saw-jan) [inline .drone.star:131]:
```suggestion
def installBrowsers():
    return [{
        "name": "install-browsers",
        "image": OC_CI_NODEJS,
        "environment": {
```

File: .drone.star [hunk 4]
```diff
@@ -89,3 +95,112 @@ def installPnpm():
...
+        ],
+    }]
+
+def e2eTests():
+    environment = {
+        "HEADLESS": "true",
+        "retry": "1",
+        "BASE_URL_OCIS": "https://ocis:9200",
+        "REPORT_TRACING": "true",
+        "REPORT_DIR": "reports/e2e",
```
Comment (saw-jan) [inline .drone.star:145]: are they implemented?
```suggestion
```

**Review by grgprarup — APPROVED** [review]
LGTM :+1:

File: .drone.star [hunk 5]
```diff
@@ -89,3 +95,112 @@ def installPnpm():
...
+def ocisService():
+    environment = {
+        "IDM_ADMIN_PASSWORD": "admin",
+        "OCIS_INSECURE": "true",
+        "OCIS_LOG_LEVEL": "error",
+        "OCIS_URL": "https://ocis:9200",
+        "PROXY_ENABLE_BASIC_AUTH": True,
+        "WEB_UI_CONFIG_FILE": "%s" % dir["webConfig"],
+        "STORAGE_HOME_DRIVER": "ocis",
+        "STORAGE_USERS_DRIVER": "ocis",
```
Comment (saw-jan) [inline .drone.star:183]:
```suggestion
```

File: .drone.star [hunk 6]
```diff
@@ -89,3 +95,112 @@ def installPnpm():
...
+                "refs/heads/main",
+                "refs/pull/**",
+            ],
+        },
+    }]
+
+def ocisService():
+    environment = {
+        "IDM_ADMIN_PASSWORD": "admin",
+        "OCIS_INSECURE": "true",
```
Comment (saw-jan) [inline .drone.star:177]:
```suggestion
        "OCIS_INSECURE": True,
```

**Review by saw-jan — APPROVED** [review]
:+1: :sparkles:

**Pull request merged**